### PR TITLE
STR-96--BE: Keep track of and send an email when student is almost out of time with a module

### PR DESCRIPTION
**Pull request by mbendandour**
Run "php artisan queue:listen --tries=1"
Open another terminal window and Run "redis-server"
Go to postman and send a user_id in the body to the route "/getReminder"
Go to your local db and edit the 'created_at" timestamp to 3 days ago
Hit send on postman

Check to see if an email was sent to you if 'smtp' is in your env files, or check your log if your email is set up to go to log

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -30,3 +30,4 @@
 
 Route::post('/verifyExcelSheet', 'AdminController@checkEmailsInJson');
 Route::post('/submitGoodParticipants','ParticipantController@addGoodParticipantsToParticipantsTable');
+Route::post('/getReminder','ModuleProgressController@remindUserofModule');
```
Comment (luisjg) [inline routes/web.php:34]: I would consider moving the api routes that the front end uses into the `routes/api.php` that way the CSRF token protection isn't needed to be modified. Or if the default Laravel configuration is used then we don't really need to explicitly tell Laravel to exclude certain routes.

File: app/Http/Middleware/VerifyCsrfToken.php [hunk 2]
```diff
@@ -20,6 +20,7 @@ class VerifyCsrfToken extends Middleware
      */
     protected $except = [
         '/loginVerification',
-        '/submitGoodParticipants'
+        '/submitGoodParticipants',
+        '/getReminder',
```
Comment (luisjg) [inline app/Http/Middleware/VerifyCsrfToken.php:24]: See comment on the `routes/web.php` file

**Review by luisjg — APPROVED** [review]
👍

**Review by liadGolan — APPROVED** [review]
👍

**Pull request merged**